python/private/coverage_deps.bzl

```diff
@@ -0,0 +1,123 @@
+        "6c4459b3de97b75e3bd6b7d4b7f0db13f17f504f3d13e2a7c623786289dd670e",
+        "aarch64-unknown-linux-gnu",
+    ),
+    (
+        "pypi__coverage_cp38_x86_64-apple-darwin",
+        "https://files.pythonhosted.org/packages/05/63/a789b462075395d34f8152229dccf92b25ca73eac05b3f6cd75fa5017095/coverage-6.5.0-cp38-cp38-macosx_10_9_x86_64.whl",
+        "d900bb429fdfd7f511f868cedd03a6bbb142f3f9118c09b99ef8dc9bf9643c3c",
+        "x86_64-apple-darwin",
+    ),
+    (
+        "pypi__coverage_cp38_x86_64-unknown-linux-gnu",
+        "https://files.pythonhosted.org/packages/bd/a0/e263b115808226fdb2658f1887808c06ac3f1b579ef5dda02309e0d54459/coverage-6.5.0-cp38-cp38-manylinux_2_5_x86_64.manylinux1_x86_64.manylinux_2_17_x86_64.manylinux2014_x86_64.whl",
+        "6b07130585d54fe8dff3d97b93b0e20290de974dc8177c320aeaf23459219c0b",
+        "x86_64-unknown-linux-gnu",
+    ),
+    (
+        "pypi__coverage_cp39_aarch64-apple-darwin",
+        "https://files.pythonhosted.org/packages/63/e9/f23e8664ec4032d7802a1cf920853196bcbdce7b56408e3efe1b2da08f3c/coverage-6.5.0-cp39-cp39-macosx_11_0_arm64.whl",
+        "95203854f974e07af96358c0b261f1048d8e1083f2de9b1c565e1be4a3a48cfc",
+        "aarch64-apple-darwin",
+    ),
+    (
+        "pypi__coverage_cp39_aarch64-unknown-linux-gnu",
+        "https://files.pythonhosted.org/packages/18/95/27f80dcd8273171b781a19d109aeaed7f13d78ef6d1e2f7134a5826fd1b4/coverage-6.5.0-cp39-cp39-manylinux_2_17_aarch64.manylinux2014_aarch64.whl",
+        "b9023e237f4c02ff739581ef35969c3739445fb059b060ca51771e69101efffe",
+        "aarch64-unknown-linux-gnu",
+    ),
+    (
+        "pypi__coverage_cp39_x86_64-apple-darwin",
+        "https://files.pythonhosted.org/packages/ea/52/c08080405329326a7ff16c0dfdb4feefaa8edd7446413df67386fe1bbfe0/coverage-6.5.0-cp39-cp39-macosx_10_9_x86_64.whl",
+        "633713d70ad6bfc49b34ead4060531658dc6dfc9b3eb7d8a716d5873377ab745",
+        "x86_64-apple-darwin",
+    ),
+    (
+        "pypi__coverage_cp39_x86_64-unknown-linux-gnu",
+        "https://files.pythonhosted.org/packages/6b/f2/919f0fdc93d3991ca074894402074d847be8ac1e1d78e7e9e1c371b69a6f/coverage-6.5.0-cp39-cp39-manylinux_2_5_x86_64.manylinux1_x86_64.manylinux_2_17_x86_64.manylinux2014_x86_64.whl",
+        "8f830ed581b45b82451a40faabb89c84e1a998124ee4212d440e9c6cf70083e5",
+        "x86_64-unknown-linux-gnu",
+    ),
+]
+#END: managed by update_coverage_deps.py script
+
+def install_coverage_deps():
+    """Register the dependency for the coverage dep.
+    """
+    for name, url, sha256, platform in _coverage_deps:
+        maybe(
+            http_archive,
+            name = name,
+            build_file_content = """
+py_library(
+    name = "coverage",
+    srcs = ["coverage/__main__.py"],
+    data = glob(["coverage/*", "coverage/**/*.py", "coverage/*.so"]),
+    exec_compatible_with = {compatible_with},
+    visibility = ["//visibility:public"],
```

Comment:
Should this be public visibility? It's really only the toolchain that should be referencing it. It'd be rather hard to use it correctly because its just a group of platform-specific files.